### PR TITLE
Add ShowHidden option to ActiveModules

### PR DESCRIPTION
**Pull request by guac420**
**Describe the pull**
This pull adds an option, ShowHidden, to the ActiveModules module that displays all enabled modules in the ActiveModules list.

**Describe how this pull is helpful**
Like most people, I try to keep my overlay clean by removing most of the modules I have enabled all the time from the ActiveModules list. But sometimes I can't remember if I have something enabled or not, and I have to search through the clickGUI to find out. The option ShowHidden displays all active modules to the ActiveModule list even if they have been toggled to not show on array. This makes it easy to see all of the active modules.

**Additional context**
I'm very bad at naming things so if anything doesn't seem to have a good name I don't care if it's changed

### CODE REVIEW

File: src/main/java/me/zeroeightsix/kami/gui/kami/theme/kami/KamiActiveModulesUI.java [hunk 1]
```diff
@@ -32,15 +32,16 @@ public void renderComponent(me.zeroeightsix.kami.gui.kami.component.ActiveModule
...
+        
         FontRenderer renderer = Wrapper.getFontRenderer();
-        List<Module> mods = MODULE_MANAGER.getModules().stream()
-                .filter(Module::isEnabled)
-                .filter(Module::isOnArray)
-                .sorted(Comparator.comparing(module -> renderer.getStringWidth(module.getName() + (module.getHudInfo() == null ? "" : module.getHudInfo() + " ")) * (component.sort_up ? -1 : 1)))
-                .collect(Collectors.toList());
+	    List<Module> mods = MODULE_MANAGER.getModules().stream()
+		    	.filter(Module::isEnabled)
+	    		.filter(Module -> (activeMods.hidden.getValue() ? 1 == 1 : Module.isOnArray()))
```
Comment (dominikaaaa) [inline src/main/java/me/zeroeightsix/kami/gui/kami/theme/kami/KamiActiveModulesUI.java:40]: Fam why is there a 1==1 lmao

**Comment by dominikaaaa** [comment]
Btw ;tsc already has a filter and when run by itself or just lists all enabled modules

File: src/main/java/me/zeroeightsix/kami/gui/kami/theme/kami/KamiActiveModulesUI.java (hunk 1 above)
Comment (guac420) [reply src/main/java/me/zeroeightsix/kami/gui/kami/theme/kami/KamiActiveModulesUI.java:40]: its a place holder I couldn't think of what else to put there

Comment (dominikaaaa) [reply src/main/java/me/zeroeightsix/kami/gui/kami/theme/kami/KamiActiveModulesUI.java:40]: `true`? or just don't put anything? I'll take a look at this on a computer later

**Comment by guac420** [comment]
> Btw ;tsc already has a filter and when run by itself or just lists all enabled modules

So would that mean that the first filter is not needed

**Comment by dominikaaaa** [comment]
No, I'm just saying you can already achieve what you're trying to do with ;tsc

But having an option to temporarily show hidden is good

File: src/main/java/me/zeroeightsix/kami/gui/kami/theme/kami/KamiActiveModulesUI.java (hunk 1 above)
Comment (guac420) [reply src/main/java/me/zeroeightsix/kami/gui/kami/theme/kami/KamiActiveModulesUI.java:40]: I can't believe I didn't think of `true` lol

**Comment by guac420** [comment]
> No, I'm just saying you can already achieve what you're trying to do with ;tsc
> 
> But having an option to temporarily show hidden is good

oh ok, I didn't know ;tsc was a thing lol

**Pull request merged**